sematic/examples/cifar_classifier/requirements.txt

```diff
@@ -4,4 +4,5 @@ torchvision
 torchmetrics
 pillow
 plotly
-ray[air]
\ No newline at end of file
+ray[air]
+sematic
```

Comment:
Maybe we should automatically include `sematic` in the build requirements, as `bazel` does.

Reply:
agreed